constitution.tex

```diff
@@ -254,7 +254,7 @@ \subsection{Modifying the Constitution}
 
 \asubsection{Active Membership Expectations}
 Active Members are expected to be active participants in CSH as defined in \ref{Expectations of Active Members}.
-Active Voting Members are expected to meet the requirements defined in \ref{Expectations of Voting Members}.
+Active Voting Members are eligible to become Voting Members if they meet the requirements defined in \ref{Qualifications of Voting Members}.
```

Comment:
This language is still confusing to me, since Voting Members is not defined (or used in this phrasing) elsewhere.

How about something like:

Active Members are eligible to vote at an Intro Eval if they meet the requirements defined in \ref{Qualifications of Voting Members}.

Reply:
Also, please see above unresolved threads.

Reply:
Voting Membership is defined in the section I'm referencing here, but I've removed the reference to Active Voting Membership.

Reply:
Specifically, what I've written is different from what you're suggesting in that voting at six weeks is an expectation of all who are eligible. That's kinda the point of this amendment, so I'll resolve this, but feel free to reopen it if you have further thoughts.